### PR TITLE
Adds section for Linux Sysctl.

### PR DESCRIPTION
**Pull request by mrunalp**
See #28 
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: config-linux.md [hunk 1]
```diff
@@ -45,6 +45,17 @@ For Linux the mapping is
...
+## Linux Sysctl
+
+```
+   "sysctl": [
+        "net.ipv4.ip_forward": "1",
+        "net.core.somaxconn": "256"
+   ],
+```
+
+sysctl modifies kernel behavior at runtime by writing to files under /proc/.
```
Comment (jonboulle) [inline config-linux.md]: Perhaps just "sysctl allows kernel parameters to be modified at runtime" - the bit about /proc seems unnecessary (and nit, /proc/sys/)

**Comment by mrunalp** [comment]
@jonboulle Done. It better matches the man page as well. Thanks!

**Comment by jonboulle** [comment]
Looks good, although now that you mention it, might as well link to the man page (like the capabilities section)

**Comment by mrunalp** [comment]
Added link to man page.

File: config-linux.md [hunk 2]
```diff
@@ -45,6 +45,17 @@ For Linux the mapping is
 
 capabilities is an array of Linux process capabilities. Valid values are the string after `CAP_` for capabilities defined in http://linux.die.net/man/7/capabilities
 
+## Linux Sysctl
+
+```
+   "sysctl": [
+        "net.ipv4.ip_forward": "1",
+        "net.core.somaxconn": "256"
+   ],
```
Comment (jonboulle) [inline config-linux.md]: trailing , here is out of vogue

**Comment by jonboulle** [comment]
lgtm

**Comment by mrunalp** [comment]
@crosbymichael PTAL. If this is good, then I will make the changes in runc.

File: config-linux.md (hunk 2 above)
Comment (mrunalp) [inline config-linux.md]: took it out

File: config-linux.md [hunk 3]
```diff
@@ -45,6 +45,17 @@ For Linux the mapping is
 
 capabilities is an array of Linux process capabilities. Valid values are the string after `CAP_` for capabilities defined in http://linux.die.net/man/7/capabilities
 
+## Linux Sysctl
+
+```
+   "sysctl": [
+        "net.ipv4.ip_forward": "1",
+        "net.core.somaxconn": "256"
+   ]
```
Comment (philips) [inline config-linux.md]: should this be a `}`? I don't think json lists can be k/v.

Comment (mrunalp) [inline config-linux.md]: @philips Good catch, fixing.

Comment (mrunalp) [inline config-linux.md]: Updated.

**Comment by philips** [comment]
lgtm

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**